server/middleware/auth.ts

```diff
@@ -29,10 +33,13 @@ export const checkUser: Middleware = async (req, _res, next) => {
 
     if (user) {
       req.user = user;
-      req.locale = user.settings?.locale
-        ? user.settings?.locale
-        : settings.main.locale;
     }
+
+    req.locale = user?.settings?.locale
+      ? user.settings.locale
+      : settings.main.locale;
+  } else {
```

Comment:
Why not just do all of this once outside of this block before the `next()`?

Reply:
Hm yeah we could do that!